packages/suite/src/components/suite/modals/index.tsx

```diff
@@ -98,8 +102,13 @@ const getDeviceContextModal = (props: Props) => {
         case 'ButtonRequest_PinEntry':
             return <ConfirmAction device={device} />;
         case 'ButtonRequest_ConfirmOutput':
-        case 'ButtonRequest_SignTx':
+        case 'ButtonRequest_SignTx': {
+            if (isCoinmarketOffers) {
+                return <CoinmarketReviewTransaction type="sign-transaction" />;
```

Comment:
i dont like this duplicates, same as duplicated signing actions

why they cant be shared?